hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py

```diff
@@ -20,6 +20,7 @@ class HrAttendanceTheoreticalTimeReport(models.Model):
     employee_id = fields.Many2one(
         comodel_name="hr.employee", string="Employee", readonly=True
     )
+    company_id = fields.Many2one(related="employee_id.company_id")
```

Comment:
Why related if you are selecting it in the query?

Reply:
Changed to leave the related field only.